### DISCUSSION TITLE
Split the button's shadow for normal and hover states

### DISCUSSION THREAD

**Discussion by parfilov** [elementor/elementor#30210] [Feature Request]
Hello!
There is a Button widget box shadow on Styles tab, but we can only set the overall shadow now.
Please make an opportunity to set shadow for normal and hover states apart.

Now I need to write custom CSS like:
```
selector .elementor-button:hover {
-webkit-box-shadow: 0 0 12px 0 rgba(0,0,0,0.12);
box-shadow: 0 0 50px 0 rgba(0,0,0,0.12);
}
```

Thanks!

**Comment by black-eye** [comment]
+1

**Comment by chrolm** [comment]
+1

**Comment by bentalgad** [comment]
+1

**Comment by brian-curtis** [comment]
+1

Was wondering why when I set a hover state box shadow on a button it added the same to the normal state...

**Comment by abdulhamidalsalman** [comment]
+1

**Comment by DmitriyDSWEB** [comment]
+1 often need add other shadow, normal, hover and active, when press button, off shadow, or change size button

**Comment by bentalgad** [comment]
+1 This is a must.

**Comment by zimmiel** [comment]
+1

**Comment by wildmercury** [comment]
+1

**Comment by soulfulwater** [comment]
This would be awesome.

**Comment by craig-tanaka** [comment]
Please get this on images also.

**Comment by 3rinyes** [comment]
Yes! +1

**Comment by LitePenguins** [comment]
Is there an update on this?

**Comment by thehervy** [comment]
+100000 this should be standard now.

I think in the next major release the elementor team should revise all components and try to make it as flexible as possible. This issue is one great example. Buttons should be able to "gain" shadow when hover - standard UX pattern, but we cannot do that without additional CSS... Please add it there in general.

**Comment by elfilalime** [comment]
+1

**Comment by cloudhal** [comment]
+1 common please, this is standard stuff.

**Comment by lindsayj23** [comment]
Has anyone figured out how to get this working without writing css for each instance?

**Comment by alexanderpankart** [comment]
Any updates on that?

**Comment by rvisima** [comment]
Seven years and still not implemented? This is such a basic thing, already implemented in other elements (like containers). Why can't we have it for buttons?

**Comment by parfilov** [comment]
> Seven years and still not implemented? This is such a basic thing, already implemented in other elements (like containers). Why can't we have it for buttons?

Because the team is working on "more important things": hosting, ai, image optimizer, email marketing. Hover shadows don't make money.

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #4859 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Don't worry, your comments, reactions, and participation in the old issue will be accounted for when considering this Feature Request for our Roadmap.

If you haven't seen our [web.fwrd 2025 event](https://www.youtube.com/live/GZgdssAaD9Y?si=rJUpSbv4h55a3BFu&t=2020), we plan to add pseudo-classes to Elementor v4 very soon. This means that you'll be able to apply different styles for `:hover`, `:focus`, and other pseudo-classes.

Happy voting! 🙂
